client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

```diff
@@ -67,11 +68,10 @@ export default function getPlanFeatures(
 	if ( isWpComBusinessPlan( productSlug ) || isWpComProPlan( productSlug ) ) {
 		return [
 			! isMonthlyPlan && freeOneYearDomain,
-			! isMonthlyPlan && liveChatSupport,
-			String( translate( 'Install custom plugins and themes' ) ),
-			String( translate( 'Drive traffic to your site with our advanced SEO tools' ) ),
-			String( translate( 'Track your stats with Google Analytics' ) ),
-			String( translate( 'Real-time backups and activity logs' ) ),
+			String( translate( 'Install Custom Plugins and themes' ) ),
+			String( translate( 'Sell products with WooCommerce' ) ),
+			String( translate( '50GB of Storage' ) ),
```

Comment:
This changes the copy for the Business plan too, if it's added to the cart by legacy Premium plan users looking to upgrade. Are we offering 50GB to new Business plan users too?

Reply:
I will double check it and circle back